specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/examples/ManagedClustersCreate_Update.json

```diff
@@ -37,7 +37,8 @@
         },
         "addonProfiles": {
         },
-        "enableRBAC": false
+        "enableRBAC": false,
+        "enablePodSecurityPolicy": true
```

Comment:
The default value of `enablePodSecurityPolicy` should be `false`. Additionally, when `enablePodSecurityPolicy` is set to `true`, it is required `enableRBAC` should also be `true`.

Reply:
Great. Thanks for the clarification. I changed the example to set enableRBAC to be true too.